lib/krane/kubernetes_resource/stateful_set.rb

```diff
@@ -65,7 +65,7 @@ def desired_replicas
     def parent_of_pod?(pod_data)
       return false unless pod_data.dig("metadata", "ownerReferences")
       pod_data["metadata"]["ownerReferences"].any? { |ref| ref["uid"] == @instance_data["metadata"]["uid"] } &&
-      @instance_data["status"]["currentRevision"] == pod_data["metadata"]["labels"]["controller-revision-hash"]
+      @instance_data["status"]["updateRevision"] == pod_data["metadata"]["labels"]["controller-revision-hash"]
```

Comment:
I'm trying to figure out whether `updateRevision` will be empty the first time a `StatefulSet` is deployed.

Reply:
I think it does the right thing:

https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/statefulset/stateful_set_control.go#L271